docs/parameters.yaml

```diff
@@ -113,6 +113,13 @@ type: string
 default: Error
 components: ["*"]
 ---
+name: Logging.File
```

Comment:
How about we go with `Logging.LogLocation` here to match other configuration options